src/gluonts/model/deepstate/_estimator.py

```diff
@@ -135,6 +137,7 @@ def __init__(
         self,
         freq: str,
         prediction_length: int,
+        cardinality: List[int],
```

Comment:
So in case one wants to turn off `use_feat_static_cat` (even if this is not recommended) what should they pass as `cardinality`? Empty list? Would it make sense to keep this `cardinality: Optional[List[int]]` with no default, but with `None` as valid value?

Reply:
I actually tried making it optional first but somehow I was able to instantiate the estimator without passing `cardinality` at all: 
`DeepStateEstimator(prediction_length=1, freq='H')`
 (although I did not explicitly make the  default equal to `None`).

It is bit strange but it looks like the `validated` decorator is passing None as default. This is not what we want because the the above default instantiation throws an exception, so I made it non-optional.

If `use_feat_static_cat == False`, then `cardinality` must be set to `[1]` explicitly, meaning all the time series belong to a single category.